sites/public/src/components/listings/search/ButtonGroup.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react"
+import { Button } from "./Button"
+import { FormOption } from "./ListingsSearchForm"
+
+type ButtonGroupProps = {
+  name: string
+  options: FormOption[]
+  value?: string
+  showBorder?: boolean
+  onChange: (name: string, value: string) => void
+}
+
+/**
+ * This component groups together multiple buttons, of which only one is
+ * selectable, and sets a value based on the active button's value when a new
+ * buttom is selected.
+ *
+ * @param props
+ * @returns
+ */
+export function ButtonGroup(props: ButtonGroupProps) {
```

Comment:
This is used for bedroom and bathroom filters

Reply:
Not sure if you have looked into this yet but at first look it seems like the existing FieldGroup component could work for this, with type button. It's already used in a few places in the partners and public sites.